public/index.html

```diff
@@ -11,6 +11,6 @@
   </head>
   <body>
     <noscript>You need to enable JavaScript to run this app.</noscript>
-    <div id="rental-assistance-finder"></div>
+    <div id="rental-assistance-finder" data-language="en"></div>
```

Comment:
Will the language be passed down by cf.gov? Just curious: why use a data attribute instead of reading from `<html lang="XX">`?

Reply:
Yes, the language will be passed in from cf.gov. There's interest from an external organization in using the Spanish version of the tool, though, and this seemed like a straightforward way for anyone adding the finder to their site to pass a language parameter to it.